templates/products/edit-product-single.php

```diff
@@ -30,21 +31,38 @@
 if ( isset( $_GET['product_id'] ) ) {
     $post_id        = intval( $_GET['product_id'] );
     $post           = get_post( $post_id );
-    $post_title     = $post->post_title;
-    $post_content   = $post->post_content;
-    $post_excerpt   = $post->post_excerpt;
     $post_status    = $post->post_status;
+    $auto_draft     = 'auto-draft' === $post_status;
+    $post_title     = $auto_draft ? '' : $post->post_title;
+    $post_content   = $auto_draft ? '' : $post->post_content;
+    $post_excerpt   = $post->post_excerpt;
```

Comment:
wouldn't it be better if we created auto-draft product via js and redirect to product edit page?
we can determine if this is a new product or not via post_status = auto-draft?